app/subscriber/src/features/manage-folder/ManageFolder.tsx

```diff
@@ -0,0 +1,178 @@
+  const navigate = useNavigate();
+  const [, { findContent }] = useContent();
+  const [folder, setFolder] = React.useState<IFolderModel>();
+  const [items, setItems] = React.useState<any>([]);
+  const [selected, setSelected] = React.useState<IContentModel[]>([]);
+
+  /** TODO: Folder content only contains contentId and sortOrder so we have to make an additional call based off of the contentIds to get the headline/summary etc..
+   * assuming we want this to differ eventually.
+   */
+  React.useEffect(() => {
+    getFolder(Number(id)).then((data) => {
+      // have to use temp not folder state as folder state will not update right away
+      const temp = data;
+      setFolder(data);
+      findContent({
+        contentIds: data.content.map((c) => c.contentId),
+      })
+        .then((data) => {
+          const tempSort = data.items.map((item) => ({
+            ...item,
+            sortOrder: temp.content.find((c) => c.contentId === item.id)?.sortOrder ?? 0,
+          }));
+          tempSort.sort((a, b) => a.sortOrder - b.sortOrder);
+          setItems(tempSort);
+        })
+        .catch();
+    });
+    // Only on initialize, or when sort order changes
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  /** function that runs after a user drops an item in the list */
+  const handleDrop = React.useCallback(
+    async (droppedItem: any) => {
+      if (!droppedItem.destination) {
+        return;
+      }
+      var updatedList = [...items];
+      // Remove dragged item
+      const [reorderedItem] = updatedList.splice(droppedItem.source.index, 1);
+      // Add dropped item
+      updatedList.splice(droppedItem.destination.index, 0, reorderedItem);
+      // Update State
+      setItems(updatedList);
+      // Update Folder
+      !!folder &&
+        (await updateFolder({
+          ...folder,
+          content: updatedList.map((item, index) => ({
+            ...item,
+            contentId: item.id,
+            sortOrder: index,
+          })),
+        })
+          .then((data) => {
+            setFolder(data);
+          })
+          .catch());
+    },
+    [folder, items, updateFolder],
+  );
+
+  /** function that will remove items from the folder when the button is clicked */
+  const removeItems = React.useCallback(async () => {
+    const updatedList = items.filter((item: any) => !selected.includes(item));
+    setItems(updatedList);
+    !!folder &&
+      (await updateFolder({
+        ...folder,
+        content: updatedList.map((item: any, index: any) => ({
+          ...item,
+          contentId: item.id,
+          sortOrder: index,
+        })),
+      })
+        .then((data) => {
+          setFolder(data);
+        })
+        .catch());
+    setSelected([]);
+  }, [folder, items, selected, updateFolder]);
+
+  /** determines whether to show body or summary text */
+  const determinePreview = (item: IContentModel) => {
+    const text = item.body ?? item.summary ?? '';
```

Comment:
This still has the issues I mentioned in the prior comment.  What we display here is based on the `content.contentType`.  We probably should have a utility function to keep it managed in one place.  Also you can't display a transcript if it hasn't been approved.

Reply:
Once you have this util function, than you need to replace all the other places we're displaying content too.